test/platform-helper.js

```diff
@@ -0,0 +1,42 @@
+const cp = require('child_process');
+const os = require('os');
+const platformHelper = require('../src/utils/platform-helper.js');
+const sinon = require('sinon');
+const tape = require('tape');
+
+let execStub;
+let platformStub;
+
+tape('platform-helper :: getIsUbuntu16 :: setup', (t) => {
+  /* eslint-disable no-tabs */
+  execStub = sinon.stub(cp, 'exec').yields(null, null, `Distributor ID:	Ubuntu
```

Comment:
Not sure if sinon yields can adjust based on what’s passed to the stub. Also, this is difficult because _src/utils/platform-helper.js_ has internal, private state.